docs/dev/events.md

```diff
@@ -16,7 +16,8 @@ In order to add a new event, follow the next steps:
 ### Event Definition
 Event definition should specify the following attributes:
 1. __name__: A unique name of the event. The name needs to remain unique and constant
-as it may be referred by the service's clients (e.g. by the UI).
+as it may be referred by the service's clients (e.g. by the UI). The name should match
+the structure <event-context>_<past_tense>.
```

Comment:
This doesn't look good when rendered

![Selection_999(461)](https://user-images.githubusercontent.com/316242/139917954-1635625e-06f3-45d5-9b7d-24244ce4770f.png)

Please add escaping or code format tags around `<event-context>_<past_tense>`

Reply:
done